src/main.cpp

```diff
@@ -2239,16 +2239,23 @@ CAmount GetBlockSubsidy(int nHeight, const Consensus::Params& consensusParams)
     }
 }
 
+static std::atomic<bool> IBDLatchToFalse{false};
+// testing-only, allow initial block down state to be set or reset
+bool TestSetIBD(bool ibd) {
+    bool ret = !IBDLatchToFalse;
```

Comment:
Note that the sense of `IBDLatchToFalse` is "backwards" -- if that boolean is `false` (its initial default constructed value), that means IBD is enabled (and vice versa).